bumble/device.py

```diff
@@ -268,6 +273,10 @@ def load_from_dict(self, config):
         self.advertising_interval_min = config.get('advertising_interval', self.advertising_interval_min)
         self.advertising_interval_max = self.advertising_interval_min
         self.keystore = config.get('keystore')
+        self.le_enabled = config.get('le_enabled', self.le_enabled)
```

Comment:
nit: align '=' characters to keep the assignments in a column form

Reply:
Done